app/services/usage_report_service.rb

```diff
@@ -0,0 +1,15 @@
+class UsageReportService
+  def self.send
+    return if ENV['DISABLE_USAGE_REPORTING']
+    Clients::Loomio.new.send_usage_report!({
+      groups_count: Group.count,
```

Comment:
Seems like we're missing a field or two here, support_email? reactions_count? Would be good to have the schema match the report that's sent over to start